monkey/monkey_island/cc/ui/src/components/pages/MapPage.js

```diff
@@ -67,6 +73,12 @@ class MapPageComponent extends AuthComponent {
               telemetryLastTimestamp: res['timestamp']
             });
           this.props.onStatusChange();
+
+          let telemConsoleRef = this.telemConsole.current;
+          if (!this.state.isScrolledUp) {
+            telemConsoleRef.scrollTop = telemConsoleRef.scrollHeight - telemConsoleRef.clientHeight;
```

Comment:
Can you, instead of calculating heights and line numbers, use `Element.scrollIntoView()` ? Wouldn't it be more trivial? https://stackoverflow.com/questions/37620694/how-to-scroll-to-bottom-in-react

Reply:
Thanks @VakarisZ . `Element.scrollIntoView()` is cool.  

We could replace the entire line ` telemConsoleRef.scrollTop = telemConsoleRef.scrollHeight - telemConsoleRef.clientHeight;` with `telemConsoleRef.scrollIntoView({behavior: "smooth"})` and also provide a animation along the way.